terraform/environments/mlra/lz-conn-test.tf

```
@@ -0,0 +1,46 @@
+locals {
+  instance-userdata = <<EOF
+#!/bin/bash
+yum install -y httpd
+systemctl start httpd
+EOF
+}
+module "ec2_instance" {
+  source                 = "terraform-aws-modules/ec2-instance/aws"
+  version                = "~> 4.0"
+  name                   = "mikereidhttptest"
+  ami                    = "ami-06672d07f62285d1d"
+  instance_type          = "t3a.small"
+  vpc_security_group_ids = [module.mikereidhttptest_sg.security_group_id]
+  subnet_id              = "subnet-06594eda5221bd3c9"
```

```suggestion
  subnet_id              = data.aws_subnet.private_subnets_a.id
```
You can use the data lookups so that you don't need to hard code vaules like subnet or vpc ids